src/js/experimental/media-captions-listbox.js

```diff
@@ -6,7 +6,7 @@ import { parseTextTracksStr, formatTextTrackObj } from '../utils/captions.js';
 import { toggleSubsCaps } from '../utils/captions.js';
 
 const captionsIndicatorInlineStyle = `
-  fill: var(--media-icon-color, #eee);
+  fill: var(--media-control-icon-color, var(--media-primary-color, rgb(238 238 238)));
```

Comment:
- Renaming `--media-icon-color` to `--media-control-icon-color` for consistency.
- Adding `--media-primary-color` for scoping and relationships of different "generic" vs. "specific" CSS vars.

Reply:
👍 `--media-control`. It's more clear than "media icon". We could also move to `--media-chrome-icon-color` (or -`--media-ui-icon-color`), since `media-control` has an unclear scope. Does media-control apply to everything or just what's in the control bar?

Reply:
Also, should the css var be called `--media-control-fill`?

Though, I think `--media-control-icon-color` and `--media-control-text-color` make sense to me.



> Does media-control apply to everything or just what's in the control bar?

yeah, that's a tricky one. Since they're global, would definitely work where ever. But the naming does make it seem like maybe it's required to be part of the control bar.